packages/cursorless-engine/src/snippets/updateSnippetBody.ts

```diff
@@ -0,0 +1,41 @@
+import { getLeadingWhitespace } from "../util/regex";
+
+export function updateSnippetBody(body: string): string {
+  // If snippet body containins tabs don't touch anything.
+  if (body.includes("\t")) {
+    return body;
```

Comment:
do we instead want to throw an error if they mix them? I think that's where we landed yesterday, but feels a bit aggressive on second thought. This is not a new api, so we could be breaking things that are working in the wild. Maybe a warning? Which would be super annoying, but as discussed, if they are mixing tabs and spaces it's probably by accident so they might appreciate knowing

Reply:
I don't want to break anything existing. If they were mixing tabs and spaces today they probably would already have problems with indentation?

Reply:
I don't feel super strongly.  If this were a new api I think I'd argue for an error, but given it's an existing api, there is an argument for maintaining the status quo when we don't know what to do.  @josharian any thoughts here?

Reply:
Blah. I'd prefer an error, but to avoid actively breaking anyone, let's make it a warning. Maybe even with a request to file an issue if see the warning, we have some idea whether this is going to be a recurring problem...

Reply:
We can't just warn if the snippet contains a tab we have to check if they are mixing; a snippet containing tab indentation is valid.

Reply:
Yep that's right. I was thinking warn if they're mixed

You should be able to detect that fairly easily when you're computing min indentation below right?

Reply:
Definitely doable